WooCommerce/Classes/ViewModels/Refunds/OrderItemRefundSummary.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+
+/// Represents a computed summary of refunded products
+///
+final class OrderItemRefundSummary {
+    let productID: Int64
+    let variationID: Int64
+
+    let name: String
+
+    /// Price, total, and totalTax are currency values.
+    /// When handling currencies, `NSDecimalNumber` is a powerhouse
+    /// for localization and string-to-number conversions.
+    /// `Decimal` doesn't have all of the `NSDecimalNumber` APIs (yet).
```

Comment:
TIL. I was going to suggest `Decimal` because it conforms to `Numeric` and you can use `+` and `sum` as well for more clarity. Which APIs are missing? Could you use `Decimal` by default and cast `as NSDecimalNumber` when you need the extra APIs?

Reply:
Good question! The short answer is: I didn't know any better at the time and I can't use Decimal here without refactoring the `CurrencyFormatter` class first.

Originally I chose `NSDecimalNumber` because:
1) The models are backed by a core data store, and the `NSManagedObject`s were `NSDecimalNumber` types. (Later I learned that I could have defined the `NSManagedObject`s to be Swift types instead.) 
2) While writing the CurrencyFormatter I came across limitations to `Decimal` (which sadly, I didn't leave my future-self enough hints about) and continued to use the `NSDecimalNumber` type in it. This could probably be re-visited. 

The [CurrencyFormatter class](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift) would need refactored to use `Decimal` before I could use `Decimal` here, because all currencies that were strings from the API have to use the [convertToDecimal](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift#L12-L27) method first. Explainer for that method, ref: p7bje6-1Ml-p2.